interfaces/cython/cantera/base.pyx

```diff
@@ -274,6 +277,13 @@ cdef class _SolutionBase:
                 self.kinetics.addReaction(reaction._reaction, False)
             self.kinetics.resizeReactions()
 
+        if not transport:
+            transport = "none"
+
```

Comment:
I think this is unnecessary -- leaving `transport` as the empty string also results in creating a base `Transport` object here.